bin/cli.js

```diff
@@ -195,6 +195,21 @@ function invoke(env) {
         .catch(exit);
     });
 
+  commander
+    .command('migrate:printAll')
+    .description('Print all migrations on directory')
```

Comment:
_in_ instead of _on_
```suggestion
    .description('Print all migrations in directory')
```